cfme/configure/settings.py

```diff
@@ -20,112 +22,191 @@ class TimeProfileAddForm(View):
     timezone = BootstrapSelect('profile_tz')
     days = BootstrapSwitch(name='all_days')
     hours = BootstrapSwitch(name='all_hours')
-    save_button = Button(VersionPick({Version.lowest(): 'Save', '5.9': 'Add'}))
-    configuration = Dropdown('Configuration')
-    table = Table("//div[@id='main_div']//table")
-    save_edit_button = Button('Save')
+    add = Button(VersionPick({Version.lowest(): 'Save', '5.9': 'Add'}))
     cancel_button = Button('Cancel')
     help_block = Text("//span[contains(@class, 'help-block')]")
 
 
-class TimeprofileAddEntities(View):
+class TimeProfileEntities(View):
+    table = Table("//div[@id='main_div']//table")
     breadcrumb = BreadCrumb()
     title = Text('//div[@id="main-content"]//h3')
 
 
-class TimeProfileAddFormView(BaseLoggedInPage):
-    timeprofile_form = View.nested(TimeProfileAddForm)
-    entities = View.nested(TimeprofileAddEntities)
+class TimeProfileView(BaseLoggedInPage):
+    form = View.nested(TimeProfileAddForm)
+    entities = View.nested(TimeProfileEntities)
     mysetting = View.nested(MySettingsView)
+    configuration = Dropdown('Configuration')
 
+    @property
+    def is_displayed(self):
+        return True if self.entities.title == 'Time Profile Information' else False
```

Comment:
**Required**

Just return the boolean result of the `self.entities.title == 'Time Profile Information'` statement, its a boolean already.